notNeededPackages.json

```
@@ -2905,6 +2905,10 @@
             "libraryName": "minimist-options",
             "asOfVersion": "4.0.1"
         },
+        "minipass": {
+            "libraryName": "minipass",
+            "asOfVersion": "3.3.1"
```

```suggestion
            "asOfVersion": "3.3.5"
```
https://unpkg.com/browse/minipass@3.3.5/index.d.ts

---

this ships with proper export